src/main/java/jenkins/plugins/office365connector/CardBuilder.java

```diff
@@ -94,8 +94,8 @@ public Card createCompletedCard(List<FactDefinition> factDefinitions) {
         return card;
     }
 
-    private String getCardThemeColor(Result result) {
-        if (result == Result.SUCCESS) {
+    private static String getCardThemeColor(Result result) {
+        if (result != null && result == Result.SUCCESS) {
```

Comment:
what is the reason of having this additional comparison?

Reply:
No reason, my mistake for keeping this in.